src/interpreter/plugin/3rdparty/jstat/jstat.ts

```
@@ -1001,3 +1001,23 @@ export function corrcoeff(arr1: number[], arr2: number[]): number {
     stdev(arr1, 1) /
     stdev(arr2, 1)
 }
+
+function arrayAscending(a: number, b: number) {
+  return a - b
+}
+
+export function percentile(arr: number[], k: number, exclusive: boolean) {
```

Does this function handle/need to handle empty array correctly?

---

> Does this function handle/need to handle empty array correctly?

Yeah it does need to handle an empty array, should throw an error. I will add that.